apps/site/src/app/admin/applicants/[uid]/components/HackerApplicantActions.tsx

```diff
@@ -31,16 +46,23 @@ function HackerApplicantActions({
 				inputMode="decimal"
 				placeholder="Applicant score"
 				step={0.5}
+				disabled={!canReview}
 			/>
 			<Button
 				onClick={() => {
 					submitReview(applicant, value);
 					setValue("");
 				}}
+				disabled={!canReview}
 			>
 				Submit
 			</Button>
 		</SpaceBetween>
+	) : (
+		<Box variant="awsui-key-label" color="text-status-warning">
+			Two reviewers have already submitted reviews. Log in as one of them to
+			submit a review.
+		</Box>
```

Comment:
Suggestion: since this component already can see the reviews, I think it'll be helpful to add the reviewers' UIDs to the warning text.